src/main/java/com/yourrents/api/property/Property.java

```diff
@@ -22,7 +22,7 @@
 
 import java.util.UUID;
 
-public record Property(UUID uuid, String name, String type, String description,
+public record Property(UUID uuid, String name, UUID typeUuid, String description,
```

Comment:
As Property-PropertyType it is a many-to-one relationship I would like to resolve it eagerly.

So it should be:

```java
public record Property(UUID uuid, String name, PropertyType type, String description,
  // etc.
```

Of course you'll need to adapt the other classes for using it.